### PR TITLE
feat: do not require a config file when starting YHS

### PR DESCRIPTION
**Pull request by dejanzele**
Closes https://github.com/G-Research/yunikorn-history-server/issues/71

### CODE REVIEW

File: internal/config/config_test.go [hunk 1]
```diff
@@ -1,8 +1,11 @@
 package config
 
 import (
```
Comment (richscott) [inline internal/config/config_test.go:3]: This doesn't break anything, but don't some/most/all IDEs sort imports in the same order as `goimports`? (stdlib packages first, then public/third-party ones, etc)

**Review by richscott — DISMISSED** [review]

File: internal/config/config_test.go (hunk 1 above)
Comment (sudiptob2) [reply internal/config/config_test.go:3]: I am using `goimports` in GoLand which sort that way.

Comment (dejanzele) [reply internal/config/config_test.go:3]: linter was broken, I pushed a fix and needed to resolve various lint issues

Comment (richscott) [reply internal/config/config_test.go:3]: 👏🏼

**Review by richscott — APPROVED** [review]

**Pull request merged**